apps/api/src/domain/models/song/commands/create-song.command.integration.spec.ts

```diff
@@ -124,6 +124,18 @@ describe('CreateSong', () => {
         });
     });
 
+    describe('when the id has not been generated via our system', () => {
```

Comment:
There should be a similar test case for every `Create` command.

Reply:
We can think about how to reuse the setup after [this story](https://www.pivotaltracker.com/story/show/182540815).